src/libs/tryResolveUrlFromApiRoot.js

```diff
@@ -1,9 +1,15 @@
 import Config from '../CONFIG';
+import * as ApiUtils from './ApiUtils';
 
 // Absolute URLs (`/` or `//`) should be resolved from API ROOT
 // Legacy attachments can come from either staging or prod, depending on the env they were uploaded by
 // Both should be replaced and loaded from API ROOT of the current environment
-const ORIGINS_TO_REPLACE = ['/+', Config.EXPENSIFY.EXPENSIFY_URL, Config.EXPENSIFY.STAGING_EXPENSIFY_URL];
+const ORIGINS_TO_REPLACE = [
+    '/+',
+    Config.EXPENSIFY.EXPENSIFY_URL,
+    Config.EXPENSIFY.STAGING_API_ROOT,
+    Config.EXPENSIFY.PRODUCTION_API_ROOT,
```

Comment:
Added the PRODUCTION_API_ROOT here to complete the list and make it clear that it is handled

I think for staging and production envs EXPENSIFY_URL == PRODUCTION_API_ROOT, but in any case now it's clearer that we handle both staging and prod cases here